spring-cloud-gcp-docs/src/main/asciidoc/config.adoc

```diff
@@ -31,6 +31,8 @@ dependencies {
 
 A https://github.com/spring-cloud/spring-cloud-gcp/tree/{git_version}/spring-cloud-gcp-samples/spring-cloud-gcp-config-sample[sample application] is available.
 
+The Google Cloud Runtime Configuration service is in beta status.
+
```

Comment:
We should mention somewhere which versions of the project it's available in -- i.e. 1.0.0.M4.
I'm not sure what the best way to do it is though. Maybe on the project page: http://cloud.spring.io/spring-cloud-gcp/.

Reply:
I'll make an issue of this